### PR TITLE
update api docs

### PR DESCRIPTION
**Pull request by ashucoder9**

### CODE REVIEW

**Comment by ashucoder9** [comment]
@meaghanfitzgerald the headers of md docs are removed to match the style of docs website remote content loader

**Comment by meaghanfitzgerald** [comment]
```md
<Callout title="Note">
The Admin API is disabled by default for security reasons. To run a node with the Admin API enabled, use [`config flag --api-admin-enabled=true`](/nodes/configure/configs-flags#--api-admin-enabled-boolean).

This API set is for a specific node, it is unavailable on the [public server](/tooling/rpc-providers).
</Callout>
```

have you tested to make sure the formatting will be applied correctly? these files are `.md`, not `.mdx`

File: api/admin/service.md [hunk 1]
```diff
@@ -110,40 +96,37 @@ curl -X POST --data '{
...
 ```
 
-Now, instead of interacting with the blockchain whose ID is
-`sV6o671RtkGBcno1FiaDbVcFv2sG5aVXMZYzKdP4VQAWmJQnM` by making API calls to
-`/ext/bc/sV6o671RtkGBcno1FiaDbVcFv2sG5aVXMZYzKdP4VQAWmJQnM`, one can also make calls to
-`ext/bc/myBlockchainAlias`.
+Now, instead of interacting with the blockchain whose ID is `sV6o671RtkGBcno1FiaDbVcFv2sG5aVXMZYzKdP4VQAWmJQnM` by making API calls to `/ext/bc/sV6o671RtkGBcno1FiaDbVcFv2sG5aVXMZYzKdP4VQAWmJQnM`, one can also make calls to `ext/bc/myBlockchainAlias`.
 
-### `admin.getChainAliases`
+### admin.getChainAliases
```
Comment (meaghanfitzgerald) [inline api/admin/service.md:111]: what were the `` removed here but not for `admin.alias`, `admin.aliasChain`, etc.?

File: api/health/service.md [hunk 2]
```diff
@@ -1,64 +1,38 @@
...
-To filter GET health checks, add a `tag` query parameter to the request.
-The `tag` parameter is a string.
-For example, to filter health results by subnetID `29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL`,
-use the following query:
+To filter GET health checks, add a `tag` query parameter to the request. The `tag` parameter is a string. For example, to filter health results by subnetID `29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL`, use the following query:
 
-```sh
+```
 curl 'http://localhost:9650/ext/health?tag=29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL'
 ```
```
Comment (meaghanfitzgerald) [inline api/health/service.md:27]: why was this formatting removed from every code block? 

```
curl 'http://localhost:9650/ext/health?tag=29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL'
```
```sh
curl 'http://localhost:9650/ext/health?tag=29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL'
```

**Comment by ashucoder9** [comment]
> ```
> <Callout title="Note">
> The Admin API is disabled by default for security reasons. To run a node with the Admin API enabled, use [`config flag --api-admin-enabled=true`](/nodes/configure/configs-flags#--api-admin-enabled-boolean).
> 
> This API set is for a specific node, it is unavailable on the [public server](/tooling/rpc-providers).
> </Callout>
> ```
> 
> have you tested to make sure the formatting will be applied correctly? these files are `.md`, not `.mdx`

yes, we parse the raw content and save a .mdx file when remote content loads, works good

File: api/health/service.md (hunk 2 above)
Comment (ashucoder9) [reply api/health/service.md]: added back. not an issue as the default formatting is picked as `bash` in new docs.

**Review by meaghanfitzgerald — APPROVED** [review]

File: api/admin/service.md [hunk 3]
```diff
@@ -1,53 +1,40 @@
...
 
 ## Format
 
-This API uses the `json 2.0` RPC format. For details, see [here](/reference/standards/guides/issuing-api-calls.md).
+This API uses the `json 2.0` RPC format. For details, see [here](/api-reference/standards/guides/issuing-api-calls).
 
 ## Endpoint
 
-```text
+```sh
```
Comment (StephenButtolph) [inline api/admin/service.md:15]: This isn't `sh` - is there a better alternative than `sh`? perhaps nothing?

File: api/admin/service.md [hunk 4]
```diff
@@ -1,53 +1,40 @@
...
 
-Assign an API endpoint an alias, a different endpoint for the API. The original endpoint will still
-work. This change only affects this node; other nodes will not know about this alias.
+Assign an API endpoint an alias, a different endpoint for the API. The original endpoint will still work. This change only affects this node; other nodes will not know about this alias.
 
-**Signature:**
+**Signature**:
 
-```text
+```sh
```
Comment (StephenButtolph) [inline api/admin/service.md:27]: This isn't `sh` - is there a better alternative than `sh`? perhaps nothing?

Comment (StephenButtolph) [reply api/admin/service.md:27]: This comment also applies to all the `Signature` blocks below.

Comment (ashucoder9) [reply api/admin/service.md]: you're right, i have removed sh from the codeblocks for endpoints and signatures.

File: api/admin/service.md (hunk 3 above)
Comment (ashucoder9) [reply api/admin/service.md]: fixed

**Review by StephenButtolph — APPROVED** [review]

**Pull request merged**